### ISSUE TITLE
p-slider Accessibility Issues

### ISSUE THREAD

**Issue by srujana10499** [primefaces/primeng#18761]
### Describe the bug

The p-slider component with checkboxes has accessibility violations when running the IBM Equal Accessibility Checker. The two issues are: Each form control must have an associated label and UI component must have aria attributes.

### Reason for not contributing a PR

- [x] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/stackblitz-starters-zyxmufqr

### Environment

stackblitz

### Angular version

20.0.4

### PrimeNG version

v20

### Steps to reproduce the behavior

1. Install the [IBM Equal Accessibility Checker](https://chromewebstore.google.com/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp)
2. Open the stackblitz
3. Run the accessibility checker
4. Filter on violations only and note the errors: 'Form control with "slider" role has no associated label' and 'Element with 'slider' role does not have the required ARIA attribute(s): 'aria-valuenow''

### Expected behavior

There are no accessibility violations for the p-slider component when slider is used
